.bazelrc

```diff
@@ -19,6 +19,7 @@ build --ios_simulator_version=15.2
 build --verbose_failures
 build --workspace_status_command=envoy/bazel/get_workspace_status
 build --xcode_version=13.2.1
+build --use_top_level_targets_for_symlinks
```

Comment:
What does this do?

Reply:
Yea so if you use bazel transitions, which the apple rules do, you get some path like `bazel-out/foo/bar-RANDOMJUNK/path/to/artifact`, with this flag it also creates the usual `bazel-bin/path/to/artifact`. It has one limitation that it doesn't work through alias targets https://github.com/bazelbuild/bazel/issues/14602 so in some cases on CI I use the real target name. Locally this shouldn't matter much since folks will just copy paste, but on CI it makes it stable